src/routes/userRoutes.ts

```diff
@@ -1,54 +1,54 @@
-import config from 'config';
+import config from '../config/index';
 import d4lRequest from '../lib/d4lRequest';
 
 const userRoutes = {
-  resolveUserId(d4lUserAlias) {
+  resolveUserId(d4lUserAlias: string) {
     const body = { value: d4lUserAlias };
     return d4lRequest.submit('POST', `${config.apiUrl()}/users/resolve`, {
       body,
     });
   },
 
-  getUserDetails(userId) {
+  getUserDetails(userId: string) {
     return d4lRequest.submit('GET', `${config.userUrl(userId)}`, {
       authorize: true,
     });
   },
 
-  fetchUserInfo(userId) {
+  fetchUserInfo(userId: string) {
     return d4lRequest.submit('GET', `${config.apiUrl()}/userinfo`, {
       authorize: true,
       ownerId: userId,
     });
   },
 
-  updateUser(userId, userData) {
+  updateUser(userId: string, userData: Object) {
```

Comment:
+1 for the `Object` type instead of `object` as the latter does not give us access to any properties of a value.